### PR TITLE
Fix EZP-23918: Empty Author not correctly detected

### PR DESCRIPTION
**Pull request by yannickroger**
Link: https://jira.ez.no/browse/EZP-23918

## Description
Message "This is empty" was not displayed for authors.

## Tests
Manual and updated unit test

### CODE REVIEW

**Comment by yannickroger** [comment]
ping @dpobel @StephaneDiot

File: Tests/js/views/fields/assets/ez-author-view-tests.js [hunk 1]
```diff
@@ -20,6 +20,21 @@ YUI.add('ez-author-view-tests', function (Y) {
...
+            },
+
+            "Test isEmpty with a filled fieldValue": function () {
+                this._testIsEmpty(
+                    this.field,
+                    false,
+                    "The field should not be seen as empty"
+                );
+            },
+
```
Comment (dpobel) [inline Tests/js/views/fields/assets/ez-author-view-tests.js]: To completely cover the added code, you should also add a test where the field contains `{fieldValue: []}`

File: Resources/public/js/views/fields/ez-author-view.js [hunk 2]
```diff
@@ -20,6 +20,19 @@ YUI.add('ez-author-view', function (Y) {
      * @extends eZ.FieldView
      */
     Y.eZ.AuthorView = Y.Base.create('authorView', Y.eZ.FieldView, [], {
+
+        /**
```
Comment (dpobel) [inline Resources/public/js/views/fields/ez-author-view.js]: Nitpick: you don't really need to copy paste the comment from the parent class

**Comment by dpobel** [comment]
Besides the comments +1

File: Tests/js/views/fields/assets/ez-author-view-tests.js (hunk 1 above)
Comment (yannickroger) [inline Tests/js/views/fields/assets/ez-author-view-tests.js]: done

**Comment by yannickroger** [comment]
I took care of @dpobel 's feedback. Ready for review

**Pull request merged**